project_template/src/operations.c

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+#include "operations.h"
+int32_t az_exampleshortname_add_two_numbers(int32_t a, int32_t b) {
+    return a + b;

```

Comment:
This would complicate the example, the intent of this function is to provide a symbol that actually ends up in the library to confirm shared libraries work (and to give something for the tests to call).

While I agree that we may want to handle overflow I don't think think it's necessary for the sample project.